src/borrower/shelf.sol

```diff
@@ -157,9 +159,11 @@ contract Shelf is Auth, TitleOwned, Math {
     // a max ceiling needs to be defined by an oracle
     function borrow(uint loan, uint currencyAmount) external owner(loan) {
         require(nftLocked(loan), "nft-not-locked");
+
         if(address(subscriber) != address(0)) {
-            subscriber.borrowEvent(loan);
+            subscriber.borrowEvent(loan, currencyAmount);
```

Comment:
Currently the only reason why we are doing this is for consistency?

Reply:
Yeah, this one is purely for consistency reasons. Wdyt, is there any downside to doing it this way?